__tests__/lib/cli/envAlias.ts

```diff
@@ -1,13 +1,20 @@
 /* eslint-disable jest/no-conditional-expect */
+/**
+ * External dependencies
+ */
+import { afterEach, describe, expect, it, jest } from '@jest/globals';
+
 /**
  * Internal dependencies
  */
-import envAlias from '../../../src/lib/cli/envAlias';
+import * as envAlias from '../../../src/lib/cli/envAlias';
 
 const parseEnvAliasSpy = jest.spyOn( envAlias, 'parseEnvAlias' );
 
 describe( 'utils/cli/envAlias', () => {
-	afterEach( parseEnvAliasSpy.mockClear );
+	afterEach( () => {
+		parseEnvAliasSpy.mockClear();
+	} );
```

Comment:
Jest's types are strictly allowing only void here so we also have to be explicit to return void-ish.